Scripts/player.gd

```diff
@@ -1,44 +1,45 @@
+class_name Player
 extends CharacterBody2D
 
 # Set Variables for overall control feel
-var move_speed : float = 100
-var jump_force : float = 200
-var gravity : float = 500
+const _move_speed : float = 100.0
+const _jump_force : float = 200.0
+const _gravity : float = 500.0
+const _death_height_y : float = 150.0
 
 # Establish the score variable and display on ui requires CanvasLayer/ScoreText nodes to be setup.
-var score : int = 0
-@onready var score_text : Label = get_node("CanvasLayer/ScoreText")
+var _score : int = 0
+@onready var _score_text : Label = get_node("CanvasLayer/ScoreText")
 
-func _physics_process(delta):
+func _physics_process(delta : float):
 	# Enable gravity and movement. I will be updating this to have mappable controls!
 	if not is_on_floor():
-		velocity.y += gravity * delta
+		velocity.y += _gravity * delta
 	velocity.x = 0
 
 	# Movement Control
-	var direction = Input.get_axis("left", "right")
+	var direction : float = Input.get_axis("left", "right")
 	if direction:
-		velocity.x = direction * move_speed
+		velocity.x = direction * _move_speed
 	else:
 		velocity.x = move_toward(velocity.x, 0, 20)
-
-
+		 
 	# Allow player to jump
 	if Input.is_action_just_pressed("jump") and is_on_floor():
-		velocity.y = -jump_force
+		velocity.y = -_jump_force
 	
 	# Engage physics engine
 	move_and_slide()
 	
 	# Fall too far and die
-	if global_position.y > 150:
+	if global_position.y > _death_height_y:
```

Comment:
did we want fall damage?

Reply:
hmm, this one is based on @Valsprites implementation I just cleaned it up, I think this is fine for now and we can specify a "death height" in some variable handled by some "level" class later

Reply:
This was mostly for testing of the basics as I was learning Godot (and still am) so fall damage is something we could do. Maybe we should re-visit this after we decide how to handle damage in general? (health bar, one hit, ect.)